src/main/java/javastart/demo/controller/ProductController.java

```diff
@@ -0,0 +1,56 @@
+package javastart.demo.controller;
+
+import javastart.demo.model.Product;
+import javastart.demo.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Controller
+public class ProductController {
+
+    private ProductRepository productRepository;
+
+    @Autowired
+    public ProductController(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    @GetMapping(value = "/products")
```

Comment:
Jeżeli jedyną wartością jaką podajesz w adnotacji jest URL to nie musisz robić tego parametru `value` tylko od razu wpisać String w adnotację (w nawiasy)